src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -165,5 +172,634 @@ ConvSolution InitAnyInvokerFactory(const ProblemDescriptionType& problem,
+
+    result.construction_params.insert(result.construction_params.end(),
+                                      {std::get<indices>(solvers).GetKernelInfo()...});
+
+    constexpr size_t buf_alignment = 256ull;
+    MultiBufferWorkspaceTraits wt({std::get<indices>(solvers).GetOutputTensorSize()...},
+                                  buf_alignment);
+
+    return std::make_tuple(TransposeInstanceTagged{
+        std::get<indices>(solvers), indices, wt, TransposeOps::CONV_OP_TAG}...);
+}
+
+template <typename CKArgsType, typename... TransposeOps>
+auto MakeTaggedTransposeInstances(ConvSolution& result,
+                                  const ExecutionContext& ctx,
+                                  const miopen::conv::ProblemDescription& problem,
+                                  const CKArgsType& ck_args,
+                                  const TransposeOps&... transpose_ops)
+{
+    return MakeTaggedTransposeInstancesHelper<CKArgsType>(
+        result,
+        ctx,
+        problem,
+        ck_args,
+        std::make_tuple(transpose_ops...),
+        std::index_sequence_for<TransposeOps...>{});
+}
+*/
+
+template <typename CKArgsType,
+          typename Input1TposeOp,
+          typename Input2TposeOp,
+          typename OutputTposeOp>
+auto MakeTaggedTransposeInstances(ConvSolution& result,
+                                  const ExecutionContext& ctx,
+                                  const miopen::conv::ProblemDescription& problem,
+                                  const CKArgsType& ck_args,
+                                  const Input1TposeOp& input1_op,
+                                  const Input2TposeOp& input2_op,
+                                  const OutputTposeOp& output_op)
+{
+
+    auto input1_solver = input1_op.MakeTransposeSolver(ctx, problem, ck_args);
+    auto input2_solver = input2_op.MakeTransposeSolver(ctx, problem, ck_args);
+    auto output_solver = output_op.MakeTransposeSolver(ctx, problem, ck_args);
+
+    // NOTE: In cases where the convolution updates only a subset of output
+    // indices, we need to first initialize the workspace buffer for
+    // output with the real tensor for the output and then apply the convolution.
+    // This is achieved by creating an input transpose op for the output workspace
+    // bufffer.
+
+    using OutputInitOp = CKTransposeInputOp<OutputTposeOp::NDIM, OutputTposeOp::CONV_OP_TAG>;
+
+    auto output_init_solver = OutputInitOp{}.MakeTransposeSolver(ctx, problem, ck_args);
+
+    result.construction_params.insert(result.construction_params.end(),
+                                      {input1_solver.GetKernelInfo(),
+                                       input2_solver.GetKernelInfo(),
+                                       output_solver.GetKernelInfo(),
+                                       output_init_solver.GetKernelInfo()});
+
+    constexpr size_t buf_alignment = 256ull;
```

Comment:
[Recommendation] Get rid of the magic number. It is used also in `GetWorkspaceSizeLayoutTransformConv()` below and maybe somewhere else.

Reply:
done.

Reply:
[resolved]